### DISCUSSION TITLE
Is it possible to embed tldraw in a non-react, or plain javascript web app?

### DISCUSSION THREAD

**Discussion by athena123** [tldraw/tldraw-v1#567] [Ideas]
It seems a really nice project. Is it possible to readily embed tldraw in a non-react, or plain javascript web app?

**Comment by steveruizok** [comment]
Hey, thanks! Not really—you'll need React to embed the Tldraw app—though remember that you can use React selectively in an app that isn't set up as a "react app". 

You could render React into a particular div in your project, and bring in the Tldraw app that way, no JSX required.

```ts
import React from 'react'
import ReactDOM from 'react-dom'
import { Tldraw } from '@tldraw/tldraw'

const rootElement = document.getElementById('myReactRoot')

ReactDOM.render(
  React.createElement(
    ReactDOM.render(
      React.createElement(Tldraw, {}, null), 
      rootElement
    )
  )
)
```

**Comment by athena123** [comment]
Thanks!! Really wish it were readily embeddable as a web component.
